src/main/resources/openapi/sqs/print-api-sqs-messaging.yaml

```diff
@@ -289,6 +289,9 @@ components:
         batchId:
           type: string
           description: The batch ID
+        printRequestCount:
+          type: integer
+          description: The number of print requests included in the batch
       required:
         - batchId
```

Comment:
Good question - I thought not because during deployment we may try processing queued messages that do not include the `printRequestCount`.
In a future release of the microservice we can make the property required.

Reply:
Ah yes, excellent point, you are right!